apis/tempo/v1alpha1/microservices_types.go

```diff
@@ -475,12 +476,12 @@ type TempoGatewaySpec struct {
 	// +kubebuilder:validation:Optional
 	TempoComponentSpec `json:"component,omitempty"`
 
-	// JaegerQuerySpec defines Jaeger Query spefic options.
+	// Gateway defines Tempo gateway spefic options.
 	//
 	// +optional
 	// +kubebuilder:validation:Optional
-	// +operator-sdk:csv:customresourcedefinitions:type=spec,displayName="Jaeger Query Settings"
-	Gateway GatewaySpec `json:"jaegerQuery"`
+	// +operator-sdk:csv:customresourcedefinitions:type=spec,displayName="Tempo Gateway Settings"
+	Gateway GatewaySpec `json:"tempoGateway"`
```

Comment:
it seems wrong to have `spec.template.gateway.tempoGateway`

Reply:
What would be your suggestion?